app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/FilterHelper.java

```diff
@@ -0,0 +1,292 @@
+import com.openclassrooms.realestatemanager.presentation.utils.customView.CustomBottomSheetRangeSlider;
+import com.picone.core.domain.entity.PointOfInterest;
+import com.picone.core.domain.entity.Property;
+import com.picone.core.domain.entity.PropertyPhoto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.openclassrooms.realestatemanager.presentation.utils.Utils.formatStringToDate;
+
+public class FilterHelper {
+
+    private BottomSheetFilterLayoutBinding bottomSheetFilterLayout;
+
+    private List<String> requestPointsOfInterests;
+    private List<String> requestPropertyType;
+    private List<PropertyPhoto> allPropertiesPhotos = new ArrayList<>();
+    private List<PointOfInterest> allPointsOfInterest = new ArrayList<>();
+    private List<Property> filteredValues;
+    private List<Property> filteredProperty;
+
+
+    public List<Property> getFilteredProperty() {
+        return filteredProperty;
+    }
+
+    public FilterHelper(BottomSheetFilterLayoutBinding bottomSheetFilterLayout) {
+        this.bottomSheetFilterLayout = bottomSheetFilterLayout;
+        initPhotoSpinner();
+        initRangeSliderValues();
+    }
+
+    private void initPhotoSpinner() {
+        List<String> numberOfPhotos = new ArrayList<>();
+        for (int i = 1; i <= 10; i++) numberOfPhotos.add(String.valueOf(i));
+        bottomSheetFilterLayout
+                .filterPropertyNumberOfPhotoSpinner.setSpinnerAdapter(numberOfPhotos);
+    }
+
+    private void initRangeSliderValues() {
+        bottomSheetFilterLayout.filterPropertyLocationPriceRangeSlider
+                .setRangeSliderValue(
+                        //Min price selectable
+                        100000
+                        //Max price selectable
+                        , 1000000
+                        //step
+                        , (float) (1000000 - 100000) / 10000);
+
+        bottomSheetFilterLayout.filterPropertyLocationSurfaceRangerSlider
+                .setRangeSliderValue(
+                        //Min area selectable
+                        100,
+                        //Max area selectable
+                        1000,
+                        //step
+                        (float) (1000 - 100) / 10);
+
+        bottomSheetFilterLayout.filterPropertyLocationRoomRangerSlider
+                .setRangeSliderValue(
+                        //Min room
+                        5,
+                        //Max room
+                        20,
+                        //step
+                        1);
+    }
+
+    //------------------------------------------FILTER--------------------------------------------------------
+
+    public void filterProperties(List<Property> allProperties) {
+        requestPointOfInterest();
+        requestPropertyType();
+        filterForLocation(allProperties);
+        filteredValues = new ArrayList<>();
+    }
+
+    private void filterForLocation(List<Property> allProperties) {
```

Comment:
The result list is not returned. 
It seems to be scoped strictly to the function.
Why ?